src/resources/elements/arrayfield.html

```diff
@@ -8,7 +8,7 @@ <h3 click.delegate="toggleCollapse()" class="title">
     </h3>
     <div class="controls" if.bind="!collapsed">
       <button class="new-item" click.delegate="addChild()">
-        New ${item.label}
+        ${newItemText ? newItemText : "New " + item.label}
```

Comment:
Remember that we will be internationalizing our UI texts. We should decide when to start adding i18n, so that we don't try to add it all at once.